web/themes/custom/accessibility_theme/src/patterns/components/hero/hero.scss

```diff
@@ -5,32 +5,78 @@
   position: relative;
 }
 
+.media-item__image {
+
+  img {
+    width: 100%;
+  }
+}
+
+.hero__content-wrapper {
+  background-color: rgba(0, 126, 154, 0.6);
+  position: relative;
+  z-index: 1;
+
+  @include breakpoint ($bp-md) {
+    background-color: transparent;
+  }
+}
+
 .hero__content {
-  background-color: $color-gray-lt;
+  background-color: $color-white;
+  color: $color-navy-blue;
+  margin: 0 auto;
   padding: 16px;
   text-align: center;
+  transform: translateY(-1.6rem);
+  width: 90%;
 
-  @include breakpoint($bp-sm) {
-    padding-bottom: 32px;
-    padding-top: 32px;
+  @include breakpoint ($bp-md) {
+    left: 20rem;
+    margin: 0;
+    margin-top: -40%;
+    max-width: 70rem;
+    padding: 32px;
+    position: absolute;
+    text-align: left;
+    transform: none;
+    width: 100%;
+  }
+
+  &::before {
+
+    @include breakpoint ($bp-md) {
+      background-color: rgba(0, 126, 154, 0.6);
+      content: '';
+      display: block;
+      height: 100%;
+      left: 0;
+      margin-left: -20rem;
+      position: absolute;
+      top: 2.4rem;
+      width: 87.5rem;
+      z-index: -1;
+    }
   }
 }
 
-.hero__heading {
+.hero__eyebrow {
+  font-size: 1.4rem;
+  font-weight: 300;
   margin-bottom: 8px;
+  text-transform: uppercase;
+}
```

Comment:
Since we'll be using the eyebrow on other components, perhaps creating a separate component for it may be better.  That way we only set one set of styles that will apply everwhere.